provider/consumer.py

```diff
@@ -213,6 +205,8 @@ def run(self):
                 if len(messages) > 0:
                     self.__fireTrigger(messages)
 
+                time.sleep(0.1)
```

Comment:
I believe this may significantly reduce the CPU usage of an idle Consumer. Without a nice little pause, this loop is always running the CPU as hard as possible.